docs/Configuring-Pantheon/Proof-of-Authority.md

```diff
@@ -0,0 +1,80 @@
+description: Pantheon Clique Proof-of-Authority (PoA) consensus protocol implementation
+path: blob/master/ethereum/core/src/main/resources/
+source: rinkeby.json
+<!--- END of page meta data -->
+
+# Proof of Authority
+
+Pantheon implements the Clique Proof-of-Authority (PoA) consensus protocol. Clique is used by the Rinkeby testnet and can be used for private networks. 
+
+In PoA networks, transactions and blocks are validated by approved accounts, known as signers. Signers take turns to create the next block. Existing signers propose and vote to add or remove signers. 
+
+## Genesis File
+
+To use Clique in a private network, Pantheon requires a PoA genesis file. When connecting to Rinkeby, Pantheon uses the [`rinkeby.json`](https://github.com/PegaSysEng/pantheon/blob/master/ethereum/core/src/main/resources/rinkeby.json) genesis file in the `/pantheon/ethereum/core/src/main/resources` directory.
+
+A PoA genesis file defines properties specific to Clique:
+
+!!! example "Genesis JSON file (stripped)"
+    ```json
+    {
+      "config": {
+        ....
+        "clique": {
+          "period": 15,
+          "epoch": 30000
+        }
+      },
+      ...
+      "extraData": "0x0000000000000000000000000000000000000000000000000000000000000000dd37f65db31c107f773e82a4f85c693058fef7a90000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000",
+      ...
+    }
+    ```
+    
+The properties specific to Clique are:
+
+* `period` - Block time in seconds. 
+* `epoch` - Number of blocks after which to reset all votes.
+* `extraData` - Initial signers are specified after the 32 bytes reserved for vanity data. 
+
+To connect to the Rinkeby testnet, start Pantheon with the [`--rinkeby`](../Reference/Pantheon-CLI-Syntax.md#rinkeby) command line option. To start a node on a PoA private network, use the [`--network-id`](../Reference/Pantheon-CLI-Syntax.md#network-id) command line option and [`--genesis`](../Reference/Pantheon-CLI-Syntax.md#genesis`) option. 
+
+### Adding and Removing Signers
+
+To propose adding or removing signers using the JSON-RPC methods, you must enable the RPC interface using the [`--rpc-enabled`](../Reference/Pantheon-CLI-Syntax.md#rpc-enabled) option. If also using the [`--rpc-api`](../Reference/Pantheon-CLI-Syntax.md#rpc-api) option, include `CLIQUE`.
+
+The JSON-RPC methods to add or remove signers are:
+
+* [clique_propose](../Reference/JSON-RPC-API-Methods.md#clique_propose)
+* [clique_getSigners](../Reference/JSON-RPC-API-Methods.md#clique_getsigners)
+* [clique_discard](../Reference/JSON-RPC-API-Methods.md#clique_discard)
+
+To propose adding a signer, call `clique_propose` specifying the address of the proposed signer and `true`.
+!!! example "JSON-RPC clique_propose request example"
```

Comment:
These look like titles in the output so I think we should probably be using title case per our doc style guide: 
JSON-RPC clique_propose request example -> JSON-RPC clique_propose Request Example

@donntrenton - what do you think?

Reply:
fixing. I just don't get this title thing about putting first letter uppercase... but I don't get it in french too ;) So I have to remind that I can't spot it directly.